ansible/roles/nginx_vhost/tasks/main.yml

```diff
@@ -146,6 +146,14 @@
   tags:
     - nginx_vhost
 
+- name: copy cors configuration if required
+  template:
+    src: "ala_cors.j2"
+    dest: "{{nginx_conf_dir}}/conf.d/ala_cors"
```

Comment:
This needs to be named including the `appname` so that we can tune it as necessary for each application and not have it interfere with other applications.

Reply:
I'll do, thanks.